cli/cmd/integration.go

```diff
@@ -221,6 +221,8 @@ func promptCreateIntegration() error {
 				"Google Container Registry (GCR)",
 				"AWS Config",
 				"AWS CloudTrail",
+				"AWS GovCloud Config",
+				"AWS GovCloud CloudTrail",
```

Comment:
To match a bit how UI shows this "dropdown menu" should we call them like this?
```suggestion
				"AWS Config (US GovCloud)",
				"AWS CloudTrail (US GovCloud)",
```

![Screen Shot 2021-05-26 at 11 43 05 AM](https://user-images.githubusercontent.com/5712253/119714408-12e28c80-be20-11eb-8d30-a5d3fd8cb9e3.png)